webapp/src/Service/ImportExportService.php

```diff
@@ -877,6 +877,14 @@ public function importAccountsJson(array $data, ?string &$message = null, ?array
             $juryTeam = null;
             $roles    = [];
             $type     = $account['type'];
+            $username = $account['username'];
+
+            $icpcRegex = "/^[a-zA-Z0-9@._-]+$/";
+            if (!preg_match($icpcRegex, $username)) {
+                $message = sprintf('Username "%s" should follow regex pattern: %s', $username, $icpcRegex);
```

Comment:
While you are testing with a regex I think it is more user friendly to talk about the allowed set of characters here.

Reply:
That takes an extra variable but now done.